rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -474,16 +544,92 @@ void gs_frame::take_screenshot(const std::vector<u8> sshot_data, const u32 sshot
 				}
 			}
 
+			screenshot_manager manager;
+			{
+				const auto fxo = g_fxo->get<screenshot_manager>();
+				std::lock_guard<std::mutex> lock(fxo->mtx);
+				manager = *fxo;
```

Comment:
What are you trying to do here? Looks like getting a copy of the whole manager.

Reply:
I didn't wanna block cellScreenshot for too long. Idk